backends/tfhe-cuda-backend/cuda/src/integer/compression/compression.cu

```diff
@@ -12,7 +12,7 @@ uint64_t scratch_cuda_integer_compress_radix_ciphertext_64(
       pbs_type, compression_glwe_dimension, compression_polynomial_size,
       (compression_glwe_dimension + 1) * compression_polynomial_size,
       lwe_dimension, ks_level, ks_base_log, 0, 0, 0, message_modulus,
-      carry_modulus, allocate_gpu_memory);
+      carry_modulus, PBS_MS_REDUCTION_T::NO_REDUCTION);
```

Comment:
Wait this was allocate_gpu_memory, it should still be there no?

Reply:
Actually the attribute in `int_radix_params` was `allocate_ms_array`. Someone (probably me) got confuse and set it using `allocate_gpu_memory`, but that was a mistake.